.github/scripts/check_newsfragments.py

```diff
@@ -23,15 +23,34 @@
 # Also ignore release branch (e.g. `2.11`) that don't have to create newsfragments
 IGNORED_BRANCHES_PATTERN = r"(master|[0-9]+\.[0-9]+)"
 
+# TODO: Remove me once the commit that correct the typo in merge to master.
+IGNORED_FRAGMENTS = [
```

Comment:
I can't see where this is used in this file...

Reply:
Should be used now, I've used the wrong constant on the `if` check

Reply:
But since the fragment was fixed on master, I don't need that skip